core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -121,6 +124,36 @@
   private SqlFunctions() {
   }
 
+  /** SQL TO_BASE64(string) function*/
+  public static String toBase64(String string) {
+    if (string == null) {
+      return null;
+    }
+    String base64 = new String(Base64.getEncoder().encode(string.getBytes(UTF_8)), UTF_8);
+    String temp = base64;
+    StringBuilder str = new StringBuilder();
+    while ((temp.length() / 76) > 0) {
```

Comment:
Does it make sense to use [Mime decoder](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html#mime) directly ? 

Can you check if MySQL produces mime version of Base64 encoding ?

Reply:
> MIME
Uses the "The Base64 Alphabet" as specified in Table 1 of RFC 2045 for encoding and decoding operation. The encoded output must be represented in lines of no more than 76 characters each and uses a carriage return '\r' followed immediately by a linefeed '\n' as the line separator. No line separator is added to the end of the encoded output. All line separators or other characters not found in the base64 alphabet table are ignored in decoding operation.

MySQL specifies that "A newline is added after each 76 characters of encoded output to divide long output into multiple lines." I haved checked the binary result of TO_BASE64 ,it seems that it not uses carriage return.